os/kernel/task/task_show_tcbinfo.c

```diff
@@ -0,0 +1,61 @@
+/****************************************************************************
+ *
+ * Copyright 2022 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <tinyara/sched.h>
+
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+
+/****************************************************************************
+ * Private Data
+ ****************************************************************************/
+
+/****************************************************************************
+ * Private Functions
+ ****************************************************************************/
+
+/****************************************************************************
+ * Public Functions
+ ****************************************************************************/
+
+/****************************************************************************
+ * Name: task_show_tcbinfo
+ *
+ * Description:
+ *   Show the specific tcb information
+ *
+ ****************************************************************************/
+
+void task_show_tcbinfo(struct tcb_s *tcb)
+{
+	lldbg("State      : %u\n", tcb->task_state);
+	lldbg("Flags      : %u\n", tcb->flags);
+	lldbg("Lock count : %u\n", tcb->lockcount);
+	lldbg("Timeslice  : %d\n", tcb->timeslice);
+	lldbg("Waitdog    : %p\n", tcb->waitdog);
+	lldbg("WaitSem    : %p\n", tcb->waitsem);
+	lldbg("MsgwaitQ   : %p\n", tcb->msgwaitq);
+	lldbg("Sigdeliver : %p\n", tcb->xcp.sigdeliver);
+	lldbg("Nsyscalls  : %u\n", tcb->xcp.nsyscalls);
+	lldbg("Syscall    : %p\n", tcb->xcp.syscall);
```

Comment:
They are lldbg not sdbg or something else. It means it's debug purpose on assert. Could you leave it at function description?

Reply:
And how about printing thread name as well? If we keep it here, we should print name, pid and priority because they are most important information. But assert already prints them. I think because of this, you don't print them. Then how about moving this in assert? Because up_taskdump is in the assert even it prints tcb infor as well.